src/pypromice/process/utilities.py

```diff
@@ -101,11 +109,9 @@ def populateMeta(ds, conf, skip):
     ds : xarray.Dataset
         L0 dataset with metadata populated as Dataset attributes
     '''
-    meta = {}
     # skip = ["columns", "skiprows"]
     for k in conf.keys():
-        if k not in skip: meta[k] = conf[k]
-    ds.attrs = meta
+        if k not in skip: ds.attrs[k] = conf[k]
```

Comment:
Looks fine and make sens to update instead of overriding all the attributes.
Are there cases where som up stream (lower level) meta data entries should be removed?

Reply:
Yes, like "format" that is first "tx" or "raw" and that needs to be updated when joined.
But in those cases I'd rather overwrite with the updated value, like:
https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/dd598a57778658acefc7a6c9278742b33f986c99/src/pypromice/process/join_l2.py#L100
or 
https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/dd598a57778658acefc7a6c9278742b33f986c99/src/pypromice/process/L2toL3.py#L34-L35

Reply:
Where are they deleted?